main.cpp

```diff
@@ -76,87 +105,145 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
     return {}; //if your function returns something other than void, add 'return {};' at the end of it.
 } 
 
-/*
- 1)
- */
-
-/*
- 2)
- */
-
-/*
- 3)
- */
-
-/*
- 4)
- */
-
-/*
- 5)
- */
-
-/*
- 6)
- */
-
-/*
- 7)
- */
-
-/*
- 8)
- */
-
-/*
- 9)
- */
-
-/*
- 10)
- */
-
-/*
- MAKE SURE YOU ARE NOT ON THE MASTER BRANCH
-
- Commit your changes by clicking on the Source Control panel on the left, entering a message, and click [Commit and push].
- 
- If you didn't already: 
-    Make a pull request after you make your first commit
-    pin the pull request link and this repl.it link to our DM thread in a single message.
-
- send me a DM to review your pull request when the project is ready for review.
-
- Wait for my code review.
- */
+// 1)
+int moveSpaceship(int positionX, int positionY)
+{
+    ignoreUnused(positionX, positionY);
+    return { };  
+}
+ 
+
+// 2_
+unsigned int makeCoffee(bool withMilk = true, bool withSugar = true)
+{
+    ignoreUnused(withMilk, withSugar);
+    return { };  
+}
+ 
+
+// 3)
+float climbLadder(float climbUp, float forHowLong)
+{
+    ignoreUnused(climbUp, forHowLong);
+    return { };  
+}
+ 
+
+// 4_
+double rideBicycle(double gearSpeed, bool lightsOn)
+{
+    ignoreUnused(gearSpeed, lightsOn);
+    return { };  
+}
+ 
+
+// 5) 
+int makeCharacter(float height, float weight)
+{
+    ignoreUnused(height, weight);
+    return { };  
+}
+ 
+
+
+// 6) 
+void answerQuestion(char isYes = 'y', char isNo = 'n')
+{
+    ignoreUnused(isYes, isNo);
+    // Shachar: no return because i used void.
+}
+ 
+
+
+// 7) 
+bool goToSleep(int forHowLong, int howDeep)
+{
+    ignoreUnused(forHowLong, howDeep);
+    return { };  
+}
+ 
+
+
+ // 8) 
+int liftWeights(bool dumbells, bool weights20K)
+{
+    ignoreUnused(dumbells, weights20K);
+    return { };  
+}
+ 
+
+
+ // 9) 
+int watchTV(int channel, float volume)
+{
+    ignoreUnused(channel, volume);
+    return { };  
+}
+ 
+
+
+ // 10) 
+int useChromeBrowser(std::string websiteAddress, bool isHTTPS)
+{
+    ignoreUnused(websiteAddress = "www.cnn.com", isHTTPS = true);
```

Comment:
don't assign values to variables when you pass them as function arguments. 
Assign values to variables BEFORE you pass them. 
And you already assigned values to them when you called `useChromeBrowser` here:
```
auto osx = useChromeBrowser("www.cnn.com", true);
```
- [ ] revise